dicttoxml.py

```diff
@@ -366,7 +372,7 @@ def convert_none(key, val, attr_type, attr={}, cdata=False):
 
 
 def dicttoxml(obj, root=True, custom_root='root', ids=False, attr_type=True,
-    item_func=default_item_func, cdata=False):
+    item_func=default_item_func, cdata=False, fold_list=True):
```

Comment:
An alternative design may be to vary the result of `item_func` for folding. What do you think about something along the lines of the following?

```python
# Using a dict may be a bit lazy here, but it should communicate the idea. You'd have 
# to check the return type in places that use the result for backward compat.
def custom_item_func(parent):
    if parent == 'Special':
        return { 'name': parent, 'should_fold': True } 
    else:
        return { 'name': parent, 'should_fold': False }

dicttoxml(…, item_func = custom_item_func)
```

This would enable you to vary the behavior based on the element or some condition. It may also simplify the number of parameters being passed about in the implementation?